sycl/doc/extensions/experimental/sycl_ext_oneapi_properties.asciidoc

```diff
@@ -169,76 +170,50 @@ namespace sycl::ext::oneapi::experimental {
 template<typename...> struct property_value;
 
 // This property has no parameters.
-struct bar_key {
-  using value_t = property_value<bar_key>;
-};
-// bar is an object of the property value type of bar.
-inline constexpr bar_key::value_t bar;
+inline constexpr property_value<struct bar_key> bar;
```

Comment:
No it's not. Also valid to do it in two lines:
```
struct bar_key;
inline constexpr property_value<bar_key> bar; //here struct isn't needed because it's already declared
```
But fort the single line version it's needed because `bar_key` needs to be declared.

Reply:
Oh, my mistake.  I thought `bar_key` was already declared, but I see now that this is the line you deleted.